app_flutter/lib/state/flutter_build.dart

```diff
@@ -58,6 +62,10 @@ class FlutterBuildState extends ChangeNotifier {
   static const String errorMessageFetchingTreeStatus =
       'An error occured fetching tree status from Cocoon';
 
+  @visibleForTesting
+  static const String errorMessageFetchingBranches =
+      'An error occured fetching branches from flutter/flutter on Cocoon.';
+
   /// Start a fixed interval loop that fetches build state updates based on [refreshRate].
   Future<void> startFetchingBuildStateUpdates() async {
```

Comment:
Nit: good to have a general function name.

Reply:
Does `startFetchingUpdates()` SG?

Reply:
SGTM.